cmd/wio/utils/config.go

```diff
@@ -13,9 +20,59 @@ func ReadWioConfig(dir string) (types.Config, error) {
     }
     ret := &types.ConfigImpl{}
     err := io.NormalIO.ParseYml(path, ret)
+
+    // check if it is a valid config
+    if ret.GetType() != constants.APP && ret.GetType() != constants.PKG {
+        return nil, errors.Stringf("wio.yml is invalid :: type => %s", ret.GetType())
```

Comment:
we don't really have this fancy punctuation in our logging anymore so why not just
`wio.yml has invalid type %s`

Reply:
it looked cool but sure I will change

Reply:
we can spice up our logging in a future release